src/models/organization.js

```diff
@@ -310,7 +310,32 @@ async function getMembersPaginated(organizationId, options) {
     perPage,
   })
 
-  return query
+  // Execute query
+  const membersPage = await query
+
+  // Query badges assigned to the users in the list
+  const userBadges = await db('user_badges')
```

Comment:
This could be turned into a badges model function, what do you think?

Reply:
Let's implement it separately, as discussed. Ticketed at https://github.com/developmentseed/osm-teams/issues/385.